contracts/tokens/EventPatch.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+// Authors: Francesco Sullo <francesco@superpower.io>
+// (c) Superpower Labs Inc.
+
+import "../SuperpowerNFT.sol";
+
+// Intermediate contract to fix a typo in the Locked and Unlocked events
+
+contract EventPatch is SuperpowerNFT {
+  // Unfortunately, there was a typo in the parameter "tokenId" of the events
+  // Locked and Unlocked, and it was spelled as "tokendId".
+  // This patch fixes the typo and emits new events with the correct parameter name.
+
+  bool public defaultLockedEmitted;
+  uint256 public lastCheckedId;
+
+  function emitDefaultLockedEvent() public onlyOwner {
+    if (!defaultLockedEmitted) {
+      emit DefaultLocked(true);
+      defaultLockedEmitted = true;
+    }
+  }
+
+  function emitNewLockedEvent() public {
```

Comment:
Questions About this Function.  We want it to be only Public?  Anyone can call a new locked event?    Also dont we handle the case where we unlock event something like ; emit Locked(i, false);